kebechet/managers/version/release_triggers.py

```diff
@@ -98,22 +120,29 @@ def adjust_version_in_sources(
             dirs[:] = [
                 d for d in dirs if not d[0] == "."
             ]  # remove hidden dirs (done in place to remove from walk)
+            if "tests" in dirs:
+                dirs.remove("tests")
             files = [f for f in files if not f[0] == "."]  # remove hidden files
-
+            files_dict = {
+                "setup.py": self.adjust_version_file,
+                "__init__.py": self.adjust_version_file,
+                "__about__.py": self.adjust_version_file,
+                "version.py": self.adjust_version_file,
+                "app.py": self.adjust_version_file,
+                "wsgi.py": self.adjust_version_file,
+                "pyproject.toml": self.adjust_version_toml_file,
+            }
             for file_name in files:
-                if file_name in (
-                    "setup.py",
-                    "__init__.py",
-                    "__about__.py",
-                    "version.py",
-                    "app.py",
-                    "wsgi.py",
-                ):
-                    file_path = os.path.join(root, file_name)
-                    adjusted_version = self.adjust_version_file(file_path)
+                if file_name in files_dict:
+                    func_to_call: Any = files_dict[file_name]
```

Comment:
could use `Callable[[str], Optional[Tuple[str, str]]]` as a type hint here. If it gives any errors, we can just leave it as callable.

Reply:
Pre commit failed with : error: Incompatible types in assignment (expression has type "Callable[[str], Optional[Tuple[Any, ...]]]", variable has type "Callable[[str], Optional[Tuple[str, str]]]")

Reply:
That's because of the function signatures of the called functions could be more descriptive too, so you would need to change this line as well as the the function signatures.